extensions/vscode/webviews/homeView/src/components/views/projectFiles/TreeProjectFiles.vue

```diff
@@ -76,6 +85,14 @@ const props = withDefaults(defineProps<Props>(), {
 const home = useHomeStore();
 const { sendMsg } = useHostConduitService();
 
+const isEntrypoint = (file: ContentRecordFile): boolean => {
+  const config = home.selectedConfiguration;
+  if (config != undefined && !isConfigurationError(config)) {
+    return file.id === config.configuration.entrypoint;
+  }
+  return false;
+};
```

Comment:
Oh, nice. It's nice that this code is closer to the actual display as well — keeps the logic closer to the thing that uses it.

I wonder why we thought we required this to come from the backend in the first place?

Reply:
> I wonder why we thought we required this to come from the backend in the first place?

I don't quite recall, but I think we have a tendency to move logic to the backend if we do (or would) repeat it a lot on the display side. This may have been one of those we made the judgement too early calls and we didn't actually need `isEntrypoint` all that much.